VR/VRInput.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "VRBase.h"
+
+// angle indexes
+#define PITCH 0
+#define YAW   1
+#define ROLL  2
+
+typedef float vec3_t[3];
+
+typedef enum ovrButton_ {
```

Comment:
I actually would like to keep it this way, it is original code from Oculus OpenXR SDK. By keeping it the same it could be easily updated if Oculus/Meta does there some updates. Is that okay?

Reply:
I actually would like to keep it this way, it is original code from Oculus OpenXR SDK. By keeping it the same it could be easily updated if Oculus/Meta does there some updates. Is that okay?

Reply:
Okay, if it's from the SDK that makes sense.

-[Unknown]